drivers/led_strip/ws2812_gpio.c

```diff
@@ -221,6 +245,21 @@ static const struct led_strip_driver_api ws2812_gpio_api = {
 			return -ENODEV;				\
 		}							\
 									\
+		for (i = 0; i < cfg->num_colors; i++) {			\
+			switch (cfg->color_mapping[i]) {		\
+			case LED_COLOR_ID_WHITE:			\
+			case LED_COLOR_ID_RED:				\
+			case LED_COLOR_ID_GREEN:			\
+			case LED_COLOR_ID_BLUE:				\
+				break;					\
+			default:					\
+				LOG_ERR("%s: invalid channel to color mapping." \
```

Comment:
Please add a space character here. Otherwise the error will be "[...] mapping.Check [...]".